libs/blocks/modal/modal.js

```diff
@@ -126,16 +126,20 @@ export async function getModal(el, custom) {
     }
   });
 
-  dialog.addEventListener('keydown', (event) => {
-    if (event.key === 'Escape') {
-      closeModals([dialog]);
-    }
-  });
 
   dialog.append(close, content);
   document.body.append(dialog);
   dialog.insertAdjacentElement('afterend', curtain);
-  firstFocusable.focus(focusVisible);
+  document.getElementsByClassName("dialog-close")[0]?.focus({ focusVisible: true });
+
+  const closeModal = (event) => {
+    var code = event.keyCode || event.which;
+    if (code === 27) {
```

Comment:
Why the change from `event.key === 'Escape'`?

Reply:
+1 . event.keyCode is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
removed deprecated keycode property and added your suggestion